protocol-units/bridge/README.md

```diff
@@ -0,0 +1,120 @@
+# Current Contract Deployments
+## Holesky Ethereum Deployments
+
+| Contract                         | Address                                      |
+|-----------------------------------|----------------------------------------------|
+| `AtomicBridgeInitiatorMove.sol`   | 0xb33d9fA868054FD3F8F02ba9Fc04B2a1862Ab855   |
+| `AtomicBridgeCounterpartyMove.sol`    | 0xA6bB0d5eA03f3bEe3510EA3Ec39e85Cd7395Bc37   |
+
+
+## testnet.movementlabs Deployments
+the movement atomic-bridge and all of its modules are deployed under the same package:
+`0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c`
+
+Verify this by running:
+```
+ movement account list --account 0xf7c46960ef7aa1b25cd5d48efef064525aa4ccec286fc3b3a7176bbfec09b84c 
+```
```

Comment:
Maybe here add something like:

Note: this step requires [Movement CLI](https://docs.movementnetwork.xyz/devs/movementcli#install-movement-cli) initialized to the following endpoints. 

You can do this by running `movement init --skip-faucet`. When prompted, enter `custom` network and the following REST endpoint:
```
https://testnet.movementnetwork.xyz
```

Your `config.yaml` should look like:

```
profiles:
  default:
    network: Custom
    private_key: "<your-private-key>"
    public_key: "<your-public-key>"
    account: "<your-account-address>"
    rest_url: "https://testnet.movementnetwork.xyz/"
```

To fund your account, visit the [faucet UI](https://faucet.movementnetwork.xyz/?network=movement) at and select Movement.